kinopoisk/movie/sources.py

```diff
@@ -281,9 +281,24 @@ class MovieTrailersPage(KinopoiskPage):
     url = '/film/{id}/video/'
 
     def parse(self):
-        trailers = re.findall(r'GetTrailerPreview\(([^\)]+)\)', self.content)
-        for trailer in trailers:
-            self.instance.add_trailer(json.loads(trailer.replace("'", '"')))
-
-        self.instance.youtube_ids = list(set(re.findall(r'//www.youtube.com/v/(.+)\?', self.content)))
-        self.instance.set_source(self.content_name)
+        # Because some films have different URL and ID
+        current_film_url = re.findall(
+            r'href="https://www.kinopoisk.ru/film/.+/video',
+            self.content
+        )
+
+        current_film_id = current_film_url[0].split('/')[-2:-1][0]
+        trailers_kinopoisk_urls = list(set(
+            re.findall(r'/film/{}/video/\d+'.format(current_film_id),
+                       self.content)
+        ))
+
+        for trailer_id in trailers_kinopoisk_urls:
+            trailer_id = trailer_id.split('/')[-1:][0]
+            self.instance.add_trailer({
```

Comment:
since there is only `id`, could you please simplify `Trailer` constructor arguments to only one required: `id`?